charts/s1-agent/values.yaml

```diff
@@ -33,10 +33,9 @@ configuration:
 secrets:
   imagePullSecret: "" # you need to specify the name of the image pull secret (created outside this chart)
   site_key:
-    # set create to "false" if you want to use a secret created outside this helm chart.
-    create: true
-    name: site_key
-    value: ""
+    create: true # set create to "false" if you want to use a secret created outside this helm chart, or for agent offline mode.
```

Comment:
it may cause problem if customer forgot to add key, maybe we should always add fake key to the k8s_automation?

Reply:
'create' is true by default, so customers don't have to do anything unless they want to use their own key, so only in this case they have to set it.